datafusion-cli/Cargo.toml

```diff
@@ -31,16 +31,11 @@ readme = "README.md"
 [dependencies]
 arrow = { version = "54.0.0" }
 async-trait = "0.1.73"
-## 1.5.13 requires a hiher MSRV 1.81 so lock until DataFusion MSRV catches up
-aws-config = "=1.5.10"
-## 1.53.0 requires a higher MSRV 1.81 so lock until DataFusion MSRV catches up
-aws-sdk-sso = "=1.50.0"
-## 1.54.0 requires a higher MSRV 1.81 so lock until DataFusion MSRV catches up
-aws-sdk-ssooidc = "=1.57.1"
-## 1.54.1 requires a higher MSRV 1.81 so lock until DataFusion MSRV catches up
-aws-sdk-sts = "=1.51.0"
-# end pin aws-sdk crates
-aws-credential-types = "1.2.0"
+aws-config = "1.5.15"
```

Comment:
Thats probably another question but do we need to follow a practice of setting the exact patch version? 
https://semver.org

We probably can provide a minor version and let rustc take the latest patch. However it is not very expected when the patch requires MSRV change though.... WDYT folks if we set all deps without the specifc patch version and see how it is going.

Reply:
> We probably can provide a minor version and let rustc take the latest patch. However it is not very expected when the patch requires MSRV change though.... WDYT folks if we set all deps without the specifc patch version and see how it is going.

Sounds good to me -- I will do so

I think the practical implication of `1.5.15` is that it will also allow anything higher, such as `1.6`, `1.5.16`

Since everything in datafusion-cli is controlled by `Cargo.lock` the versions won't change without a change to that file anyways. 

That being said there is no reason to have this patch version as you say so I will remove it